app/code/Magento/Sales/Block/Adminhtml/Order/Create/Form/Account.php

```diff
@@ -204,4 +206,17 @@ private function extractValuesFromAttributes(array $attributes): array
 
         return $formValues;
     }
+
+    /**
+     * Retrieve email is required field for admin order creation
+     *
+     * @return bool
+     */
+    private function isEmailRequiredCreateOrder()
```

Comment:
I think `isEmailRequiredToCreateOrder` is more suitable name.

Reply:
@swnsma 

Hi,
The function name is changed to isEmailRequiredToCreateOrder.